apps/Dataman/src/screens/stockIssue.tsx

```diff
@@ -0,0 +1,150 @@
+import { StyleSheet, ScrollView, View } from "react-native";
+import React, { useState } from "react";
+import NavBar from "../components/NavBar";
+import { Button, InputItem, List, Text } from "@ant-design/react-native";
+import { AntDesign } from "@expo/vector-icons";
+
+const StockIssue = () => {
+  const [quantity, setQuantity] = useState<string>();
+  const [rate, setRate] = useState<string>();
+  const [remarks, setRemarks] = useState<string>();
+  return (
+    <View>
+      <NavBar />
+      <ScrollView>
+        <List style={{ marginBottom: 10 }}>
+          <List.Item>
+            <Text
+              style={{
+                fontSize: 28,
+                fontWeight: "600",
+                paddingVertical: 5,
+              }}
+            >
+              Stock Issue Entry
```

Comment:
refactor this into an array containing list data and use .map() to iteratate over this list in order to create the list view